src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -706,6 +706,21 @@ public CheckContextSummary ApplyGetContextSummary()
 
             return summary;
         }
+
+        /// <summary>
+        /// Used to check if given FirstNameNode(Variable) node is a valid variable or not.
+        /// </summary>
+        public bool IsNodeInvalid(FirstNameNode node)

```

Comment:
>IsNodeInvalid [](http://example.com/codeflow?start=20&length=13)

nit: for code reading purposes, would be better to make this method resulting in a positive answer e.g. `IsNodeValid`?